Client/Frontend/Widgets/PhotonActionSheetProtocol.swift

```diff
@@ -113,6 +113,7 @@ extension PhotonActionSheetProtocol {
     
     func getTabActions(tab: Tab, buttonView: UIView,
                        presentShareMenu: @escaping (URL, Tab, UIView, UIPopoverArrowDirection) -> Void,
+                       presentToolsMenu: @escaping ([[PhotonActionSheetItem]]) -> Void,
```

Comment:
I think we can avoid this By doing.

```
let toolsAction = PhotonActionSheetItem(title: Strings.AppMenuToolsTitleString, iconString: "menu-Tools", accessory: .Disclosure) { _ in
self.presentSheetWith(title: "Tools", actions: [[findInPageAction, toggleDesktopSite, pinToTopSites]], on: presentableVC, from: buttonView)
}
```

inside `getTabActions` removing it from BVC.